contracts/hello-tacos.storageList.mligo

```diff
@@ -4,10 +4,10 @@
 // Define a storage variable. By default, the first variable in this list will be used to deploy
 let store42tacos: storage = {
     available_tacos = 42n;
-    admin = ("tz1ge3zb6kC5iUZcXsjxiwwtU5MwP37T6m1z" : address)
+    admin = ("tz1VSUr8wwNhLAzempoch5d6hLRiTh8Cjcjb" : address)
```

Comment:
We probably want to let users know that they should set change this accordingly. Or not? Not sure